README.org

```diff
@@ -0,0 +1,112 @@
+  #+BEGIN_SRC sh
+    conda install micronota
+  #+END_SRC
+
+
+  Or you can install through ~pip~:
+  #+BEGIN_SRC sh
+    pip install micronota
+  #+END_SRC
+
+  To install the latest developping version:
+  #+BEGIN_SRC
+    pip install git+git://github.com/biocore/micronota.git
+  #+END_SRC
+
+* Prepare Databases
+
+  To prepare (download and format) the files of TIGRFAM to the right form read by micronota:
+  #+BEGIN_SRC
+    micronota database prepare tigrfam --cache_dir ~/database
+  #+END_SRC
+
+* Config File
+
+  By default, micronota will read =~/.micronota.config= file to set up the environment or tune the parameters, if this config file exists.
+
+  For example, the default directory to store the database files is =~/micronota_db=, but you can override it to =/home/username/db= by setting this in =~/.micronota.config=:
+  #+BEGIN_EXAMPLE
+    [DEFAULT]
+    db_path = /home/username/db
+  #+END_EXAMPLE
+
+  micronota will look for the key =db_path= in the section =DEFAULT= to update the database path.
+
+  Besides setting up the environment, you can also specify the parameter for each individual tools. For example, if you want to run Prodigal with genetic translation table 1, instead of the default translation table, you can create a new file run.cfg (although you can also just add into =~/.micronota.config=):
+  #+BEGIN_EXAMPLE
+    [DEFAULT]
+    # overwrite the default setting
+    db_path = /home/username/another_db
+
+    [prodigal]
+    # set translation table to 1
+    -t = 1
+  #+END_EXAMPLE
+
+  Here, Prodigal has an option =-t= to specify translation table, so you set =-t= to =1=. All the options of all the supported tools should be able to be set up this way.
+
+  After creating the config file, then you can run:
+  #+BEGIN_SRC sh
+    micronota --config run.cfg annotate -i input.fa
+  #+END_SRC
+
+* Print Configure Info
+
+  To check the micronota setup, you can run:
+  #+BEGIN_SRC
+    micronota info
+  #+END_SRC
+
+  It will print out the system info, databases available, external tools, and other configuration info.
+
+* Sequence Features to Identify
+
+    | Features                | Supported | Tools                                            |
+    |-------------------------+-----------+--------------------------------------------------|
+    | coding gene             | yes       | Prodigal                                         |
+    | tRNA                    | yes       | Aragorn                                          |
+    | ncRNA                   | yes       | Infernal                                         |
+    | CRISPR                  | yes       | MinCED                                           |
+    | ribosomal binding sites | ongoing   | RBSFinder                                        |
+    | prophage                | ongoing   | PHAST                                            |
+    | replication origin      |           | Ori-Finder 1 (bacteria) & Ori-Finder 2 (archaea) |
```

Comment:
means "plan to add"

On Fri, Feb 19, 2016 at 2:28 PM, Daniel McDonald <notifications@github.com>
wrote:

> In README.org
> <https://github.com/biocore/micronota/pull/44#discussion_r53527934>:
>
> > +    micronota info
> > +  #+END_SRC
> > +
> > +  It will print out the system info, databases available, external tools, and other configuration info.
> > +
> > +* Sequence Features to Identify
> > +
> > +    | Features                | Supported | Tools                                            |
> > +    |-------------------------+-----------+--------------------------------------------------|
> > +    | coding gene             | yes       | Prodigal                                         |
> > +    | tRNA                    | yes       | Aragorn                                          |
> > +    | ncRNA                   | yes       | Infernal                                         |
> > +    | CRISPR                  | yes       | MinCED                                           |
> > +    | ribosomal binding sites | ongoing   | RBSFinder                                        |
> > +    | prophage                | ongoing   | PHAST                                            |
> > +    | replication origin      |           | Ori-Finder 1 (bacteria) & Ori-Finder 2 (archaea) |
>
> what does a blank entry here mean?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/micronota/pull/44/files#r53527934>.
>